pyproject.toml

```diff
@@ -100,6 +100,10 @@ force-exclude = '''
 ^/test-data
 '''
 
+[tool.codespell]
+ignore-words-list = "ans,ccompiler,corus,fo,froms,haa,hax,notin,ot,statics,whet,zar"
```

Comment:
2.3.0 added inline ignores. We should probably use those for single occurrences like `froms`.
```py
# codespell:ignore froms
```